docs/tutorials/index.md

```diff
@@ -27,9 +27,9 @@ This is a selected subset of Gluon tutorials that explains basic usage of Gluon
 - [Fast, portable neural networks with Gluon HybridBlocks](http://gluon.mxnet.io/chapter07_distributed-learning/hybridize.html)
 - [Training on multiple GPUs with gluon](http://gluon.mxnet.io/chapter07_distributed-learning/multiple-gpus-gluon.html)
 
-## MXNet
+## Low Level Interface
```

Comment:
MXNet Symbolic API?

Reply:
Thinking further this page may be better organized overall. For instance before beginning with gluon, we can have an introductory para about high level interface (gluon) and the low level interface (symbolic). And explicitly call out that gluon is now the preferred way to way to use MXNet. Once that is established the further sections may get more obvious while improving the flow of the page. Adding @aaronmarkham for any further comments.

Reply:
Yes, I'm reorganizing this page. I'll update the PR shortly.